### ISSUE TITLE
Automagically scroll code panel

### ISSUE THREAD

**Issue by nielmclaren** [furiousgreencloud/PermanentDeviation#7]
In the replay, coding is hard to follow right now because it's often outside of the currently visible area in the code text area. Automagically scroll to see the place where the coding is happening.

**Comment by furiousgreencloud** [comment]
Or how about colour coding the text to illustrate it's age?

maybe we could use an off the self syntax hi-lighter and then tweet it to
show the age of the code?

-b

**Comment by nielmclaren** [comment]
Oh. My. Gawds. The code itself could be art. This is a cool idea. Maybe background colour of blocks of text will show age and we'll also have background colour of lines.

Hmm, interesting side-thought. Initially, I wanted the tweet to show a screenshot but what if it showed a snippet of code-change?

**Comment by furiousgreencloud** [comment]
There are some very beautiful code diffing tools out there, (
https://github.com/kpdecker/jsdiff) the diffs of the session could be
tweets out....

**Comment by desiringproductions** [comment]
Awesome... whatever suits your programmer fancy.
